docs/source/frequently_asked_questions.ipynb

```diff
@@ -752,6 +752,18 @@
     "feature_matrix.head()"
    ]
   },
+  {
```

Comment:
can you add a separate question that demonstrates autonormalize on the same dataset as above?

Reply:
In the example above, they use a normalized part of the customer dataset (just the transaction dataframe). Should I just demonstrate it on the whole customer table as a single table? And should I change the example above to use the whole dataset as a single table?